src/ZendeskApi.Client/Resources/IJobStatusResource.cs

```diff
@@ -0,0 +1,37 @@
+using System.Threading.Tasks;
+using ZendeskApi.Client.Models;
+using ZendeskApi.Client.Responses;
+
+namespace ZendeskApi.Client.Resources
+{
+    public interface IJobStatusResource
+    {
+        #region List
```

Comment:
Do we have `#region`s in the other resources as well? ([Regions Considered Harmful.](https://softwareengineering.stackexchange.com/questions/53086/are-regions-an-antipattern-or-code-smell/53178))